From b1f4fa1401cba2e359e5a4b3ea2bafd119fca62b Mon Sep 17 00:00:00 2001
From: Pino Toscano <pino@kde.org>
Date: Tue, 24 Apr 2018 06:30:19 +0200
Subject: oscar: include buffer.h

This header uses Buffer as by-value parameter, so make sure it knows
about it.

BUG: 393372
FIXED-IN: 18.04.1
---
 protocols/oscar/liboscar/tasks/messagereceivertask.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/protocols/oscar/liboscar/tasks/messagereceivertask.h b/protocols/oscar/liboscar/tasks/messagereceivertask.h
index 8f52cd7..908e903 100644
--- a/protocols/oscar/liboscar/tasks/messagereceivertask.h
+++ b/protocols/oscar/liboscar/tasks/messagereceivertask.h
@@ -21,6 +21,7 @@
 #include <QByteArray>
 #include "oscarmessage.h"
 #include "oscartypeclasses.h"
+#include "buffer.h"
 
 class QTextCodec;
 
-- 
cgit v0.11.2

